src/movies.js

```diff
@@ -1,16 +1,212 @@
 // Iteration 1: All directors? - Get the array of all directors.
+function getAllDirectors(movies) {
+    let allDir = movies.map((elem) => {
+        return elem.director;
+    })
```

Comment:
Awesome! using arrow functions like a boss. You can even reduce the syntax more and make this solution in one line! ;)

Reply:
`return movies.map(elem => elem.director)`